examples/shiny_syntax/fileInput.R

```diff
@@ -0,0 +1,19 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- shinyUI(
+  semanticPage(
+    fileInput("file_ex", "File Input", width = "400px", type = "small"),# numericinput specific params
```

Comment:
this comment should not be here `# numericinput specific params`?

Reply:
Done